raiden/tests/unit/pack.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { LockedTransfer, packMessage } from 'raiden/messages';
+
+describe('packMessage', () => {
+  test('LockedTransfer', () => {
+    /* eslint-disable @typescript-eslint/camelcase */
+    const message = {
```

Comment:
Maybe it's useful to anotate the type `message: LockedTransfer`, so we get a compile-type warning if structure changes, otherwise for tsc it's just an `object` which can have any schema, and the error will be captured only at test-time.

Reply:
I actually tried to do this but tsc would show errors be because it expected a `PositiveNumber` instead of a `number`. My guess would be that I would have to do something similar to the fold for every field. If you have any suggestion on how I would avoid doing this I would gladly accept it.